spec/integration/machine_spec.rb

```diff
@@ -52,6 +52,29 @@
           source_dest_check: false
         ).and be_idempotent
       end
+
+      it "machine with from_image option is created from correct image", :super_slow do
```

Comment:
oh derp, you're totally right - we're testing the machine `from_image` spec.  Going to get more coffee now!